specification/operationalinsights/data-plane/Microsoft.OperationalInsights/preview/2021-05-19_Preview/OperationalInsights.json

```diff
@@ -123,6 +123,78 @@
         }
       }
     },
+    "/{resourceId}/query": {
```

Comment:
While we don't have an explicit convention for this, I would be tempted to put the new route under x-ms-paths since openapi-adhering tools will not be able to make use of the new operation (due to the disabling of url encoding of the parameter value)

Reply:
@rakshith91 Reminder to discuss with Johan. Development on this new Azure Monitor Query SDK feature is blocked until this PR merges.